client/src/app/services/token-interceptor.ts

```diff
@@ -37,9 +37,10 @@ export class TokenInterceptor implements HttpInterceptor {
     // If token is available in the cookie
     if (token) {
       // We set the Authorization header: token inside the object for every request
-      headersConfig['Authorization'] = `Token ${token}`;
+      const authHeader = 'Authorization';
+      headersConfig[authHeader] = `Token ${token}`;
```

Comment:
you are doing that because the lint is complaining?

Reply:
Yes it was complaining locally. After this fix it stopped but now it is complaining from the workflow. I dont know why.

Reply:
Perfect 👍